### ISSUE TITLE
Hold duration for the end of a set of subtitles, and subtitles for the H.E.V. suit (and potentially others).

### ISSUE THREAD

**Issue by Rozekail** [hzqst/MetaHookSv#512]
I'm not sure if it's a bug or not, but subtitles don't display for H.E.V. suit dialogue, despite being in the dictionary file and seemingly properly called by the game.

Additionally, I cant seem to find a way to make subtitles stay on-screen longer but ONLY if they're the last to be displayed in a set of subtitles. For example, "If you see one of these buttons, press it, and I will appear to demonstrate certain moves." The last line displayed it just "certain moves.", which means they fade out IMMEDIATELY, leaving no time to really read them if you aren't already looking. But setting hold durations affects ALL subtitles, not just the last line displayed. But again, I'm not sure if that's a bug, or if it should be a feature request.

**Comment by hzqst** [comment]
You can utilize GitHub's fork feature by forking a copy of the MetaHookSv source code -> clone the forked repository to your local machine -> commit your subtitle changes to the \Build\svencoop\captionmod\dictionary_english.txt and Build\svencoop\captionmod\dictionary.csv files in your local source code directory -> submit your changes to GitHub -> initiate a Pull Request on GitHub, and then I can merge your changes into the main codebase. This way, your changes will be bundled into the release package the next time the MetaHookSv releases.

Here's a 1-minute video that can quickly guide you through the steps: https://www.youtube.com/watch?v=8lGpZkjnkt4

As for the issue of subtitles not displaying despite the audio files being configured, I suggest opening an issue specifically for that; in this way, I can promptly locate and fix the problem.

**Comment by hzqst** [comment]
Added in https://github.com/hzqst/MetaHookSv/releases/tag/v20241130a, which affects only the last line.

![image](https://github.com/user-attachments/assets/6c6794b7-ea89-4741-9c7c-84c87ef5a4f7)

**Comment by Rozekail** [comment]
Was going to mark this as closed, but forgot it also mentions the H.E.V. issue. Any idea on what's causing H.E.V. subtitles to not be displayed? I loaded one instance and tried to debug and the only relevant info I found was:

CaptionMod: SENTENCE [!HEV_A0] found. dist: 3.97, avol: 0.25
CaptionMod: Sound [fvox/bell.wav] not found.
CaptionMod: Sound [fvox/_comma.wav] not found.
CaptionMod: Sound [fvox/HEV_logon.wav] not found.

So it seems like it finds the captionmod data just fine but thinks it can't find the relevant sounds (even though those sounds definitely play properly in-game)? I don't really know.
